extensions/interactions/NumericInput/directives/numeric-input-validation.service.spec.ts

```diff
@@ -337,9 +336,9 @@ describe('NumericInputValidationService', () => {
 
   it('should generate errors in the given input', () => {
     expect(validatorService.getErrorString(1200000000E+27, false)).toEqual(
```

Comment:
getErrorStringI18nKey

In general, be careful about distinguishing the raw strings and the i18n keys.